File: frontend/src/app/dto/learning-material/learning-material.insert.req.dto.ts
export class LearningMaterialInsertReqDto {

    private learningCode! : string
	private learningSubject! : string
	private learningDescription! : string
	private classId! : number
	private fileCode! : string
	private fileExt! : string

    public get _learningCode(): string {
        return this.learningCode;
    }

    public set _learningCode(_learningCode: string) {
        this.learningCode = _learningCode;
    }

    public get _learningSubject(): string {
        return this.learningSubject;
    }

    public set _learningSubject(_learningSubject: string) {
        this.learningSubject = _learningSubject;
    }

    public get _learningDescription(): string {
        return this.learningDescription;
    }

    public set _learningDescription(_learningDescription: string) {
        this.learningDescription = _learningDescription;
    }

    public get _classId(): number {
        return this.classId;
    }

    public set _classId(_classId: number) {
        this.classId = _classId;
    }

    public get _fileCode(): string {
        return this.fileCode;
    }

    public set _fileCode(_fileCode: string) {
        this.fileCode = _fileCode;
    }

    public get _fileExt(): string {
        return this.fileExt;
    }

    public set _fileExt(_fileExt: string) {
        this.fileExt = _fileExt;
    }

}

File: backend/src/main/java/com/lawencon/elearn/dto/learningmaterial/LearningMaterialInsertReqDto.java
package com.lawencon.elearn.dto.learningmaterial;

public class LearningMaterialInsertReqDto {
	private String learningCode;
	private String learningSubject;
	private String learningDescription;
	private Long classId;
	private String fileCode;
	private String fileExt;
	public String getLearningCode() {
		return learningCode;
	}
	public void setLearningCode(String learningCode) {
		this.learningCode = learningCode;
	}
	public String getLearningSubject() {
		return learningSubject;
	}
	public void setLearningSubject(String learningSubject) {
		this.learningSubject = learningSubject;
	}
	public String getLearningDescription() {
		return learningDescription;
	}
	public void setLearningDescription(String learningDescription) {
		this.learningDescription = learningDescription;
	}
	public Long getClassId() {
		return classId;
	}
	public void setClassId(Long classId) {
		this.classId = classId;
	}
	public String getFileCode() {
		return fileCode;
	}
	public void setFileCode(String fileCode) {
		this.fileCode = fileCode;
	}
	public String getFileExt() {
		return fileExt;
	}
	public void setFileExt(String fileExt) {
		this.fileExt = fileExt;
	}
}
